user/serializers.py

```diff
@@ -7,7 +8,14 @@ class Meta:
         model = get_user_model()
         fields = ("id", "email", "password", "is_staff")
         read_only_fields = ("is_staff",)
-        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
+        extra_kwargs = {
+            "password": {
+                "write_only": True,
+                "min_length": 5,
+                "style": {"input_type": "password"},
+                "label": gettext("Password"),
+            }
+        }
```

Comment:
The `extra_kwargs` for the `password` field in `UserSerializer` are correctly set to ensure the password is write-only, has a minimum length, and is styled as a password input.